pkg/models/cedar_system.go

```diff
@@ -22,7 +22,6 @@ type CedarSystem struct {
 	UUID                    zero.String `json:"uuid"`
 }
 
-// CedarSubSystem is the model for a sub system that comes back from the CEDAR Core API
```

Comment:
Why was this comment removed? (Not that it's the greatest comment to begin with, but just curious!)

Reply:
I honestly can't remember, but I want to say it's because it was basically the same info you get from the name of the struct.